root/release/edit/tracklist.tt

```diff
@@ -328,6 +325,26 @@
        %]
     </table>
 
+    <!-- ko if: hasTooEarlyFormat -->
+      <div class="warning">
+        [%- warning_icon %]
+        <p>
+          <strong>[% add_colon(l('Warning')) %]</strong>
+          [% l('This medium is set to a format that did not exist on the selected release date.') %]
+          <br />
+          <!-- ko if: hasUnconfirmedEarlyFormat -->
+          [% l('This should never be correct. Make sure that, say, your release set to CD isn’t meant to be a vinyl, or your vinyl isn’t actually a shellac disc.') %]
+          <!-- /ko -->
+          <!-- ko ifnot: hasUnconfirmedEarlyFormat -->
+          [% l('You didn’t cause this problem right now, so you don’t need to do anything about it. But if you have some time, consider trying to find out what the real format should be!') %]
```

Comment:
IMHO the most common case is likely that the date for the earliest release event actually matches the original release date for that release group. In such case, the format is usually correct but the release event should be moved to a separate release. So I would rather suggest to look at possibly different releases, roughly:

```suggestion
          [% l('You didn’t cause this problem right now, so you don’t need to do anything about it. But if you have some time, consider trying to find out what the real format should be or if the release should be split!') %]
```